spec/support/equipment_model_helper.rb

```diff
@@ -0,0 +1,47 @@
+require 'spec_helper'
+
+module EquipmentModelHelper
```

Comment:
Yes, I think that there will be separate helpers for each model. I'll probably create a folder for all mock helpers, and I won't replicate the mocking methods across helpers; this one will eventually use `CategoryHelper#mock_category`